cypress/support/commands.js

```diff
@@ -23,7 +23,34 @@
 //
 // -- This will overwrite an existing command --
 // Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+import faker from 'faker';
+import './commands';
+Cypress.Commands.add('login', () => {
+  cy.task('generateUser').then((user) => {
+    const { email, username, password } = user;
 
+    cy.request('POST', '/api/users', {
+      user: {
+        email,
+        username,
+        password
+      }
+    }).then(response => {
+      const user = {
```

Comment:
The variable name 'user' is redeclared which may lead to confusion. Consider renaming the variable.